src/models/Offer.js

```diff
@@ -54,6 +55,7 @@ export type OfferRequest = {
   provider: string,
   fromAssetAddress: string,
   toAssetAddress: string,
+  trackId: string,
```

Comment:
Do we have it for fiat offers as well? 🤔

Reply:
I don't think so. The changes show it as in Fiat, but the param is added to OfferRequest.

Reply:
Yeah, sorry, mislooked! All good here 👍